src/components/dashboard/ReceiveByQR.web.js

```diff
@@ -31,25 +34,21 @@ const ReceiveByQR = ({ screenProps }) => {
         log.debug({ url })
 
         if (url === null) {
-          store.set('currentScreen')({
-            dialogData: {
-              visible: true,
-              title: 'Error',
-              message: 'Invalid QR Code. Probably this QR code is for sending GD',
-              dismissText: 'Ok',
-            },
+          showDialog({
+            title: 'Error',
+            message: 'Invalid QR Code. Probably this QR code is for sending GD',
+            dismissText: 'Ok',
+            type: 'error',
```

Comment:
As of `showErrorDialog` implementation, we can pass the `message` to it and avoid using all the extra definition that is already set in `showErrorDialog`.

I mean, change this to:
```javascript
showErrorDialog('Invalid QR Code. Probably this QR code is for sending GD.')`
```

Reply:
Excellent! this is much beautiful this way